library/src/main/java/org/codemc/worldguardwrapper/WorldGuardWrapper.java

```diff
@@ -27,13 +27,20 @@ public static WorldGuardWrapper getInstance() {
     private Listener eventListener;
 
     private WorldGuardWrapper() {
+        String version;
         try {
             Class.forName("com.sk89q.worldguard.WorldGuard");
-            delegate = new org.codemc.worldguardwrapper.implementation.v7.WorldGuardImplementation();
```

Comment:
Why don't we keep it like that?

Reply:
it lets us handle the exceptions correctly